pages/index.tsx

```diff
@@ -233,7 +237,10 @@ const MostPopularPlugins: React.FC = () => (
 
 // declare react component for All Plugins section
 const AllPlugins: React.FC = () => (
-  <Box sx={{ flexGrow: 1 }}>
+  <Box sx={{
+    flexGrow: 1,
+    marginBottom: 2
```

Comment:
Added bottom margin (as it felt unnatural to have the cards flat against the bottom of the page)